ui/component/transactionListTable/internal/transaction-list-item.jsx

```diff
@@ -28,7 +28,7 @@ class TransactionListItem extends React.PureComponent<Props> {
     if (type === TXN_TYPES.TIP) {
       return <Button button="secondary" icon={ICONS.UNLOCK} onClick={this.abandonClaim} title={__('Unlock Tip')} />;
     }
-    return <Button button="secondary" icon={ICONS.DELETE} onClick={this.abandonClaim} title={__('Abandon Claim')} />;
+    return <Button button="secondary" icon={ICONS.DELETE} onClick={this.abandonClaim} title={__('Abandon Support')} />;
```

Comment:
This isn't correct. We still want to show "Abandon Claim" in some cases.

Reply:
Can you explain to me in which case should be used Abandon claim and Abandon support?

Reply:
if "type === TXN_TYPES.SUPPORT"

Reply:
fixed, thanks